### PR TITLE
[IT] "Term" template, gender regex, more tests

### PR DESCRIPTION
**Pull request by atti84it**
Many little things:
-support for "Term" template
-gender regex
-"debolmente" test updated
-added tests for "lettore" and "condividere"

### CODE REVIEW

File: wikidict/lang/it/__init__.py [hunk 1]
```diff
@@ -48,7 +50,10 @@
 templates_italic: Dict[str, str] = {}
 
 # Templates more complex to manage.
-templates_multi: Dict[str, str] = {}
+templates_multi: Dict[str, str] = {
+    # {{Term|statistica|it}} -> <small>(<i>statistica</i>)</small>
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:54]: No need to be so specific ;)
```suggestion
    # {{Term|statistica|it}}
```

File: tests/test_it.py [hunk 2]
```diff
@@ -7,6 +7,21 @@
 @pytest.mark.parametrize(
     "word, pronunciations, genre, etymology, definitions",
     [
+        (
+            "condividere",
+            ["kondiˈvidere"],
+            "",
+            [
+                "dal latino <i>cum</i> e <i>dividere</i>; l'attuale uso improprio del verbo <i>condividere</i> è dovuto alla diffusione dei social network negli anni 2000 e 2010"
```
Comment (BoboTiG) [inline tests/test_it.py:15]:
```suggestion
                "dal latino <i>cum</i> e <i>dividere</i>; l'attuale uso improprio del verbo <i>condividere</i> è dovuto alla diffusione dei social network negli anni 2000 e 2010". # noqa
```

File: tests/test_it.py [hunk 3]
```diff
@@ -16,6 +31,17 @@
...
         ),
+        (
+            "lettore",
+            ["letˈtore"],
+            "m",
+            ['dal latino <i>lector</i>, derivazione di <i>legĕre</i> ossia "leggere"'],
+            [
+                "chi legge un libro, un giornale o una rivista",
+                "<small><i>(religione)</i></small> persona che in alcune chiese cristiane, come la Chiesa cattolica, la Chiesa anglicana e quella ortodossa, è incaricata di proclamare la parola di Dio e altri testi nelle celebrazioni liturgiche e di esercitare altri compiti in campo pastorale",
+                "<small><i>(elettronica)</i></small> <small><i>(informatica)</i></small> <small><i>(tecnologia)</i></small> <small><i>(ingegneria)</i></small> dispositivo elettronico che decodifica e riceve informazioni da un supporto",
```
Comment (BoboTiG) [inline tests/test_it.py:42]:
```suggestion
                "<small><i>(religione)</i></small> persona che in alcune chiese cristiane, come la Chiesa cattolica, la Chiesa anglicana e quella ortodossa, è incaricata di proclamare la parola di Dio e altri testi nelle celebrazioni liturgiche e di esercitare altri compiti in campo pastorale",  # noqa
                "<small><i>(elettronica)</i></small> <small><i>(informatica)</i></small> <small><i>(tecnologia)</i></small> <small><i>(ingegneria)</i></small> dispositivo elettronico che decodifica e riceve informazioni da un supporto",  # noqa
```

File: tests/test_it.py [hunk 4]
```diff
@@ -7,6 +7,21 @@
...
+            ["kondiˈvidere"],
+            "",
+            [
+                "dal latino <i>cum</i> e <i>dividere</i>; l'attuale uso improprio del verbo <i>condividere</i> è dovuto alla diffusione dei social network negli anni 2000 e 2010"
+            ],
+            [
+                "spartire con altri",
+                "avere qualcosa in comune con qualcun altro",
+                "essere d'accordo con altri su un punto di vista",
+                "<small><i>(filosofia)</i></small> <small><i>(economia)</i></small> mettere spazi e risorse in comune con altri",
```
Comment (BoboTiG) [inline tests/test_it.py:21]:
```suggestion
                "<small><i>(filosofia)</i></small> <small><i>(economia)</i></small> mettere spazi e risorse in comune con altri",  # noqa
```

**Review by BoboTiG — COMMENTED** [review]
Great start :)

You mention '"debolmente" test updated', but it seems not. Forget something?

**Comment by atti84it** [comment]
> Great start :)
> 
> You mention '"debolmente" test updated', but it seems not. Forget something?

Ow no. Just confused. Files are ok. Commit comment is wrong.

**Comment by BoboTiG** [comment]
Could you fix errors reported by the "Code quality" job? I suggested modifications already ;)

File: wikidict/lang/it/__init__.py [hunk 5]
```diff
@@ -48,7 +50,10 @@
 templates_italic: Dict[str, str] = {}
 
 # Templates more complex to manage.
-templates_multi: Dict[str, str] = {}
+templates_multi: Dict[str, str] = {
+    # {{Term|statistica|it}}
+    "Term": "small(term(parts[1]))",
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:55]:
```suggestion
    "Term": "small(term(parts[1]))",
    "term": "small(term(parts[1]))",
```

File: tests/test_it.py [hunk 6]
```diff
@@ -7,6 +7,21 @@
 @pytest.mark.parametrize(
     "word, pronunciations, etymology, definitions",
     [
+        (
+            "condividere",
+            ["kondiˈvidere"],
+            "",
+            [
+                "dal latino <i>cum</i> e <i>dividere</i>; l'attuale uso improprio del verbo <i>condividere</i> è dovuto alla diffusione dei social network negli anni 2000 e 2010". # noqa
```
Comment (BoboTiG) [inline tests/test_it.py:15]:
```suggestion
                "dal latino <i>cum</i> e <i>dividere</i>; l'attuale uso improprio del verbo <i>condividere</i> è dovuto alla diffusione dei social network negli anni 2000 e 2010",  # noqa
```

File: wikidict/lang/it/__init__.py [hunk 7]
```diff
@@ -4,6 +4,9 @@
 # Regex to find the pronunciation
 pronunciation = r"{IPA\|/([^/]+)/"
 
+# Regex to find the gender
+genre = r"{{Pn\|?w?}} ''([fm])[singvol ]*''"
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:8]:
```suggestion
gender = r"{{Pn\|?w?}} ''([fm])[singvol ]*''"
```

**Pull request merged**